### ISSUE TITLE
Review Documentation images to make sure all images have ALT txt tags 

### ISSUE THREAD

**Issue by bcarramolino** [Qiskit/documentation#1659]
### URL to the relevant documentation

I noticed that some images have missing ALT text tags. Becky Dimock and Jennifer Janecheck shared the following documentation about image accessibility for reference> https://www.ibm.com/able/toolkit/design/content/#alternative-text-for-visuals. Could you please review the Documentation pages to make sure all images have Alt text? Thank you for your help!

### Select all that apply

- [ ] typo
- [ ] code bug
- [ ] out-of-date content
- [ ] broken link
- [X] other

### Describe the fix.

Providing descriptive tags to images when required per the IBM Accessibility Guidelines https://www.ibm.com/able/toolkit/design/content/#alternative-text-for-visuals
